packages/i18next/src/lib/types.ts

```diff
@@ -83,3 +92,8 @@ export interface InternationalizationContext {
 export interface InternationalizationClientOptions {
 	i18n?: InternationalizationOptions;
 }
+
+export interface I18nextFormatters {
+	name: string;
+	format(value: any, lng: string | undefined, options: any): string;
```

Comment:
Is options really any?

Reply:
Yup :(

Reply:
Unbelievable... :/

Reply:
btw, for the curious: https://github.com/i18next/i18next/blob/master/index.d.ts#L755

Reply:
It's because when using [nesting](https://www.i18next.com/translation-function/nesting) it is entirely possible that `value` isn't just a `string`. Because if the nested property was passed into another formatter it could be basically `any`thing. Objects, arrays, booleans, integers, you name it.